tests/acceptance/test_organization_sentry_app.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import absolute_import
+
+from exam import mock
+
+
+from sentry.testutils import AcceptanceTestCase
+from tests.acceptance.page_objects.organization_integration_settings import (
+    OrganizationIntegrationSettingsPage
+)
+
+
+class OrganizationSentryAppAcceptanceTestCase(AcceptanceTestCase):
+    """
+    As a user(type?), I can setup, configure, and remove a sentry app.
```

Comment:
This test seems like it's for the installing and uninstalling, and not setup/configure/removing a sentry app